### PR TITLE
fix(config): update Zooz ZEN15 Config to FW 1.6+

### LINKED ISSUE #3745: Updated device configuration: Zooz Zen15 firmware 1.06 parameter 27
Hello,

Amazing work here, I'm way too new to this but wanted to bring up a request to modify a parameter to the following configuration:

https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x027a/zen15.json

According to the manufacturer:  https://www.support.getzooz.com/kb/article/616-zen15-power-switch-ver-1-06-advanced-settings/

If the firmware version is 1.06 or greater, parameter 27 gets a few additional/tweaked settings.

Remaining the same as what exists in the database:
0 = Always On

Changed/New:
1 = Changes from 'Momentarily (at status change)' to 'LED displays power consumption color when on, none when off' (per the LED Notification section in the attached User Manual) 
2 = (New) 'LED displays power consumption color for 5 seconds regardless if on or off' (again per the LED Notification section in the attached User Manual)
3 = (New) 'LED indicator is OFF'

I was unsure if it made sense to create a new zen15.json file with this firmware version (as done with the zen16.json files in https://github.com/zwave-js/node-zwave-js/tree/master/packages/config/config/devices/0x027a), or if these changes could be incorporated into the existing zen15.json using 

"$if": "firmwareVersion >= 1.06",

And how that might affect <1.06 and the description change for setting 1 in this parameter.

I also was unsure how to apply the style guide for the description of the changed or new parameters to be consistent, I certainly appreciate the fact that you focus a lot of effort on user consistency.  Overall, I was just too unsure as to how to make the appropriate change, but would really love to see the additional parameters (especially since I prefer to have the LED indicators off).    Hoping this provides enough information so someone more skilled and experienced than me could incorporate these changes.  Thanks and keep up the great work!

### PR DESCRIPTION
**Pull request by GrizzlyAK**
This PR updates Parameter #27 to include additional options for later FW versions. Note that the changes to this parameter were introduced in FW v1.3, not v1.6 as suggested in the Issue (per the device's [changelog](https://www.support.getzooz.com/kb/article/309-zen15%C2%A0power-switch-change-log/)).

For reference, the manufacturer's parameter guide is [here](https://www.support.getzooz.com/kb/article/616-zen15-power-switch-ver-1-06-advanced-settings/).

Also fixed an incorrect FW version used in an existing conditional for Parameters #33 and #34. v1.6  (v1.06) is the latest FW, whereas v2.4 refers to HW Version 2.04 (which introduced FW 1.6).

fixes: #3745

### CODE REVIEW

**Comment by GrizzlyAK** [comment]
Semanticized PR title...

**Comment by GrizzlyAK** [comment]
Clearly, I don't have GitHub figured out yet. Not sure why it says I'm asking for 13 commits when the first 10 were merged  several days ago. Let me know if there is something else I need to do. ☹️

File: packages/config/config/devices/0x027a/zen15.json [hunk 1]
```diff
@@ -123,6 +124,35 @@
...
+			"label": "LED Indicator",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED indicator will display power consumption whenever the device is plugged in (LED stays on at all times, even if the switch is off)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen15.json]:
```suggestion
					"label": "Always On",
```

File: packages/config/config/devices/0x027a/zen15.json [hunk 2]
```diff
@@ -123,6 +124,35 @@
...
+					"value": 0
+				},
+				{
+					"label": "LED indicator will display power consumption whenever the device is ON and will turn off when the switch is OFF",
+					"value": 1
+				},
+				{
+					"label": "LED indicator will display the level of power consumption for 5 seconds only whenever the device is turned on or off (LED indicator will stay off for most of the time)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen15.json]:
```suggestion
					"label": "Momentary (at status change)",
```

File: packages/config/config/devices/0x027a/zen15.json [hunk 3]
```diff
@@ -123,6 +124,35 @@
...
+					"value": 2
+				},
+				{
+					"label": "LED indicator is OFF at all times except for inclusion, exclusion, and Z-Wave range test)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen15.json]:
```suggestion
					"label": "Always off (except for inclusion, exclusion, and Z-Wave range test)",
```

File: packages/config/config/devices/0x027a/zen15.json [hunk 4]
```diff
@@ -123,6 +124,35 @@
...
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED indicator will display power consumption whenever the device is plugged in (LED stays on at all times, even if the switch is off)",
+					"value": 0
+				},
+				{
+					"label": "LED indicator will display power consumption whenever the device is ON and will turn off when the switch is OFF",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen15.json]:
```suggestion
					"label": "On when switch is on; Off when switch is off",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Lets shorten the labels a bit

**Review by AlCalzone — APPROVED** [review]
OK with my changes

**Comment by GrizzlyAK** [comment]
@AlCalzone, just for clarity, I retained the manufacturer's wording, albeit lengthy, because in v1.3 they completely changed the function _and_ behavior of the LED to not only indicate power on/off, but to provide color-coded feedback as to actual power consumption by the device controlled, i.e., Cyan=300-600 W, Yellow=900-1200W, etc., The longer labels clarified that _power consumption_ was being displayed at all times, except for option 3.

I have 5 of these devices and the feature is pretty handy, especially since they can control high-power devices.

**Comment by AlCalzone** [comment]
Ahh I see. We should be able to explain that shorter than the original labels though.

**Pull request merged**